tpm/aziot-tpmd/src/error.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#[derive(Debug)]
+pub enum Error {
+    Internal(InternalError),
+    InvalidParameter(Option<(&'static str, Box<dyn std::error::Error + Send + Sync>)>),
+}
+
+impl Error {
+    // pub(crate) fn invalid_parameter<E>(name: &'static str, err: E) -> Self
```

Comment:
Remove commented-out code, and presumably the `InvalidParameter` variant entirely since you're not using it.